### ISSUE TITLE
Doc builds are failing on quite a few PRs

### ISSUE THREAD

**Issue by asinghvi17** [MakieOrg/Makie.jl#2823]
I've seen this in #2811, #2822, and #2818.

```
→ evaluating code [example_8731089[887](https://github.com/MakieOrg/Makie.jl/actions/runs/4562075546/jobs/8048980920?pr=2811#step:6:888)802825557] in (tutorials/scenes.md)        Killed
```
is the last line of the error.  Is it possible that we're hitting some kind of time or rate limit on Github Actions?

**Comment by SimonDanisch** [comment]
Ah hm -.- I think that's because `record_states` in the new JSServe version treats the sliders as dependent, leading to many more states to record, taking forever to render out...Although on my pc it was somewhat ok (a minute or so?), but who knows how long CI will take ^^

**Comment by SimonDanisch** [comment]
Fixed in #2428
